src/components/Board.js

```diff
@@ -4,14 +4,28 @@ import Square from './Square';
 import PropTypes from 'prop-types';
 
 
-const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
+const generateSquareComponents = (squares, onClickCallback, player) => {
 
+  const squareComponents = squares.map((row) => {
+    return row.map(({id, value}) => {
+  return (
+    <Square
+       id={id}
+       value={value}
+       onClickCallback={onClickCallback}
+       player={player}
```

Comment:
This is not a big issue but I wanted to point out that player is being passed as a prop but isn't ever used by the `Square` component. Its good practice to remove any props that you end up not using (with the exception of `key`). 
This is similar to the advice we'd give about arguments in functions.